common.blocks/forum-form/forum-form.js

```diff
@@ -0,0 +1,35 @@
+modules.define('forum-form', ['i-bem__dom', 'jquery'],
+    function(provide, BEMDOM, $) {
+
+provide(BEMDOM.decl(this.name, {
+    onSetMod: {
+        js: {
+            inited: function() {
+                var textarea = this.findBlockInside('textarea'),
+                    plainTextButton = this.findBlocksInside('button')[0],
```

Comment:
this is not reliable way to find needed blocks. it'd be much better to mix elements of `forum-form` and find blocks like this:
```js
plainTextButton = this.findBlockInside('plain', 'button'),
previewButton = this.findBlockInside('preview', 'button');
```

which means «find button on my own element `plain`» (or `preview` element).